release-notes/release-notes/5.8-release-notes.md

```diff
@@ -43,25 +81,75 @@ You can now pull specific data from REST API response headers using .NET regex c
 
 ![](../../.gitbook/assets/connections-functions/ConnectionsSchemaWarning.png)
 
-### Addition of Listener section in triggered Sources for MongoDB Collection (Cinchy Event Triggered) and REST API (Cinchy Event Triggered) Sources
+#### Addition of Listener section in triggered Sources for MongoDB Collection (Cinchy Event Triggered) and REST API (Cinchy Event Triggered) Sources
+
+- We added a Listener section to the MongoDB Collection (Cinchy Event Triggered)
+  and REST API (Cinchy Event Triggered) Sources. You can now manage the event
+  trigger within the Connections UI. This reduces the complexity of managing the
+  Listener Config table.
+
+#### DataPollingConcurrencyIndex improvements
+
+We improved the implementation of `DataPollingConcurrencyIndex`. Additionally,
+we added logging around the **Acquire** and **Release** processes to enhance
+monitoring.
+
+#### Model support for Cinchy Table Target
 
-- We added a Listener section to the MongoDB Collection (Cinchy Event Triggered) and REST API (Cinchy Event Triggered) Sources. You can now manage the event trigger within the Connections UI. This reduces the complexity of managing the Listener Config table.
+Model support is now available in the Cinchy Table Target within Connections.
+This feature offers plasticity without the need for defining or calling
+additional APIs on Cinchy.
+
+#### Enhanced JSON Handling for Kafka destinations
+
+Kafka destinations now support more robust JSON field handling, allowing for nested
+values within generated Kafka messages. Additionally, data with invalid JSON
+formats will be caught and rejected without causing the sync to fail. This
+allows for more complex data structures to be accurately represented while also
+providing a fail-safe for invalid JSON data. Cinchy now supports the following
+features:
+
+- **Nested JSON Support**: Convert JSON in source columns to nested values in
+  Kafka messages.
+- **Invalid JSON Handling**: Reject rows with invalid JSON without failing the
+  sync.
```

Comment:
The final implementation is a little different, as follows.

```suggestion
#### Connections - JSON content type for text columns

When configuring a connection source with text columns, it is possible to specify a JSON content type. This instructs the system to interpret the contents as a JSON object and pass it through as such. This is useful when the target (e.g. Kafka) supports and expects a JSON object for a specific target column. When setting this option, the value should always be valid JSON.

Alternatively, the original, default behaviour of treating text columns as plaintext is unchanged. As plaintext, the contents of the column will be passed through as a string, even if it could be interpreted as JSON.
```

Reply:
Got it. I will commit and might make minor editorial changes. Thanks @RyanNieuwoudt